CHANGE_LOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## Changed
+- GH-92: Made the metric installations data counts are recorded to configurable per deployment
```

Comment:
this doesn't quite read as a valid english sentence :)

Reply:
No... no it does not.